src/idiomatic/foundations-api-design/predictable-api/common-traits/copy.md

```diff
@@ -0,0 +1,47 @@
+---
+minutes: 10
+---
+
+# Copy
+
+Like `Clone`, but indicates the type is can be bitwise copied.
+
+Derivable: ✅ When to implement: Sometimes.
```

Comment:
I would change "Sometimes" to "If possible" or something similarly strong. From [`When _should_ my type be `Copy`?](https://doc.rust-lang.org/stable/std/marker/trait.Copy.html#when-should-my-type-be-copy):

> Generally speaking, if your type can implement Copy, it should. Keep in mind, though, that implementing Copy is part of the public API of your type. If the type might become non-Copy in the future, it could be prudent to omit the Copy implementation now, to avoid a breaking API change.

This is what I teach when I reach the section on copy and move semantics in Rust Fundamentals and I would like this reference section to be consistent with that.